lib/mrsk/commands/healthcheck.rb

```diff
@@ -11,14 +11,15 @@ def run
       "--label", "service=#{container_name}",
       "-e", "MRSK_CONTAINER_NAME=\"#{container_name}\"",
       *web.env_args,
+      *web.health_check_args,
       *config.volume_args,
       *web.option_args,
       config.absolute_image,
       web.cmd
   end
 
-  def curl
-    [ :curl, "--silent", "--output", "/dev/null", "--write-out", "'%{http_code}'", "--max-time", "2", health_url ]
+  def status
+    pipe container_id, xargs(docker(:inspect, "--format", DOCKER_HEALTH_STATUS_FORMAT))
   end
```

Comment:
Healthcheck and App both define a `status` method to fetch container status. It's a bit same-y, but the container name/id differs in each, and letting them figure that out themselves ended up looking clearer.